packages/typeorm/package.json

```diff
@@ -29,16 +29,16 @@
     "url": "https://github.com/ckb-js/kuai/issues"
   },
   "dependencies": {
+    "inversify": "6.0.1",
     "reflect-metadata": "0.1.13",
-    "inversify": "^6.0.1",
-    "rxjs": "^7.5.7",
-    "typeorm": "0.3.10",
-    "tslib": "2.4.1"
+    "rxjs": "7.5.7",
+    "tslib": "2.4.1",
+    "typeorm": "0.3.10"
   },
   "devDependencies": {
     "@jest/globals": "29.3.1",
-    "typescript": "4.8.4",
     "ts-node": "10.7.0",
-    "pg": "^8.8.0"
+    "typescript": "4.8.4",
+    "pg": "8.8.0"
```

Comment:
`pg` should be listed in the `dependencies` as it's the default database used in kuai for now. It will be removed or listed in `devDependencies` when other databases are supported.

Reply:
> `pg` should be listed in the `dependencies` as it's the default database used in kuai for now. It will be removed or listed in `devDependencies` when other databases are supported.

tests used it

Reply:
> `pg` should be listed in the `dependencies` as it's the default database used in kuai for now. It will be removed or listed in `devDependencies` when other databases are supported.

OK